apps/omg_watcher/test/eventer/core_test.exs

```diff
@@ -28,11 +28,10 @@ defmodule OMG.Watcher.Eventer.CoreTest do
   @tag fixtures: [:alice, :bob]
   test "notify function generates 2 proper address_received events", %{alice: alice, bob: bob} do
     recovered_tx =
-      API.TestHelper.create_recovered(
-        [{1, 0, 0, alice}, {2, 0, 0, bob}],
-        API.Crypto.zero_address(),
-        [{alice, 100}, {bob, 5}]
-      )
+      API.TestHelper.create_recovered([{1, 0, 0, alice}, {2, 0, 0, bob}], API.Crypto.zero_address(), [
```

Comment:
This doesn't look like `formatter` change - make sure you run formatter on this file

Reply:
It was the formatter.